src/releasers/go-yoshi.ts

```diff
@@ -176,7 +176,7 @@ export class GoYoshi extends ReleasePR {
       changelogEntry,
       updates,
       version: candidate.version,
-      includePackageName: this.monorepoTags,
+      includePackageName: false,
```

Comment:
Just making sure this is expected to be here (this change is not listed in the description).